content/vault/v1.21.x (rc)/content/api-docs/system/internal-counters.mdx

```diff
@@ -1292,6 +1292,13 @@ it may be up to 20 minutes delayed.
 - This endpoint can be called from all namespaces. The requested namespace will act as a filter. The exported data will include activity for the requested
   namespace and all of its children.
 
```

Comment:
Why are we documenting the response parameters before we've provided the endpoint path and request parameters?

Reply:
will add response fields after sample request